lib/datahub-client/data_platform_catalogue/client/search.py

```diff
@@ -422,7 +422,7 @@ def _parse_container(
         """
         tags = parse_tags(entity)
         terms = parse_glossary_terms(entity)
-        last_modified = parse_last_modified(entity)
+        last_modified = parse_metadata_last_ingested(entity)
```

Comment:
there is inconsistency here too but I'm not sure we're doing anything with this value in the frontend